README.rst

```diff
@@ -76,23 +76,23 @@ Using Twine
 
    .. code-block:: console
 
-       $ python setup.py sdist bdist_wheel
+      $ python setup.py sdist bdist_wheel
 
-2. Upload with ``twine`` to `Test PyPI`_ and verify things look right. Twine
-will automatically prompt for your username and password:
+2. Upload with ``twine`` to `Test PyPI`_ and verify things look right.
+   Twine will automatically prompt for your username and password:
```

Comment:
I suspect this change, and this change alone, fixes the rendering issue that you're attributing to indentation.

Reply:
There are two issues with the screenshot:
1. (2) is not rendered as an enumerate: this is fixed by the diff you quoted
2. The codeblock shows as quoted: this is fixed by aligning the nested element correctly.

Reply:
@McSinyx 

> (2) is not rendered as an enumerate: this is fixed by the diff you quoted

This only happens in 1 of the 2 places you have screencaptures of, so it's a problem with GitHub which has never fully embraced or supported the Python community or its standards in the first place.

> The codeblock shows as quoted: this is fixed by aligning the nested element correctly.

Given that literally every other nested block shows appropriately in this document, I'm not inclined to agree with your assessment here.

Reply:
Actually neither is rendered as enumerate, but I was wrong about the second point I think.